frontend/ui/wikipedia.lua

```diff
@@ -1531,6 +1531,7 @@ abbr.abbr {
                     no_compression = false
                 end
                 epub:add("OEBPS/"..img.imgpath, content, no_compression)
+                logger.dbg("Adding OEBPS/"..img.imgpath)
```

Comment:
Besides this line, which I thought seemed like a decent idea, all of the other changes effectively revert the NewsDownloader copy to be identical to the original again.

Reply:
Don't you have enough info about the image in the download code ? Does seeing `Adding OEBPS/img0003.jpg` brings anything useful ?
I don't mind having it, but it feels like if we logger.dbg adding an image to the ZIP, we should have logger.dbg for all the other items added to the zip - which are 6 other files that are always added, so not really useful.

Reply:
Indeed, I'd overlooked the `success`. In that case I'm not really sure why any of the differences from the original existed. Oh well.